core/src/test/scala/io/finch/OutputSpec.scala

```diff
@@ -83,4 +93,56 @@ class OutputSpec extends FinchSpec {
         Encode[String, Text.Plain].apply(op.value, op.charset.getOrElse(StandardCharsets.UTF_8))
     }
   }
+
+  it should "create an empty endpoint with given status when calling unit" in {
+    check { s: Status =>
+      val rep = Output.unit(s).toResponse[Text.Plain]
+      rep.status === s
+    }
+  }
+
+  it should "throw an exception on calling value on an Empty output" in {
+    import org.scalatest.prop.PropertyChecks
+    PropertyChecks.forAll { e: Output.Empty =>
+      the[IllegalStateException] thrownBy e.value should have message "empty output"
```

Comment:
Why using `forAll` directly here? Is that b/c of `the[Exception]` syntax? If we could do just wrap `e.value` with `Try` and test the equality instead.